scripts/publish.js

```diff
@@ -46,6 +47,28 @@ if (!publishers.includes('salesforce')) {
   exit(-1);
 }
 
+// Checks that you have specified the next version as an environment variable, and that it's properly formatted.
+const nextVersion = process.env['SALESFORCEDX_VSCODE_VERSION'];
+if (!nextVersion) {
+  console.log(
+    'You must specify the next version of the extension by setting SALESFORCEDX_VSCODE_VERSION as an environment variable.'
+  );
+  exit(-1);
+} else {
+  const [version, major, minor, patch] = nextVersion.match(
```

Comment:
Should we do a try catch here? If the version is malformed it fails with a cryptic error `TypeError: Cannot read property 'Symbol(Symbol.iterator)' of null`

Reply:
@vazexqi

Reply:
No, it's OK. This is a script meant for a limited people to execute. It just needs to fail fast. Errors that are bit cryptic are OK.